packages/openapi-v3-types/src/openapi-v3-spec-types.ts

```diff
@@ -21,6 +22,10 @@ export type OpenApiSpec = OAS3.OpenAPIObject;
 // tslint:disable-next-line:no-any
 export type ExtensionValue = any;
```

Comment:
AFAICT, `ExtensionValue` type is no longer used, could you please remove it too?

Reply:
It's still referenced in our code base to avoid `any`.

Reply:
See https://github.com/strongloop/loopback-next/pull/1271